website/docs/components/tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,147 @@
+## How to use this component
+
+The HDS tooltip is provided as both a `TooltipButton` component and as an `hds-tooltip` Ember [modifier](/components/tooltip?tab=code#ember-modifier). To be accessible, the element which triggers a tooltip needs to be an interactive element which can receive focus. This means that:
+
+* if the tooltip needs to be applied to a non-interactive element, the `TooltipButton` component should be used as it wraps the opener element with a `button` element.
+* if the tooltip needs to be applied to an element which is already interactive, the `hds-tooltip` modifier should be used.
+
+Currently, the tooltip uses [Tippy.js](https://atomiks.github.io/tippyjs/) under the hood for its main functionality.
+
+<!-- use the same heading order from Guidelines -->
+### TooltipButton
+
+!!! Warning
+
+As the `TooltipButton` component wraps its content with an HTML `<button>` element, there may be possible layout changes when using it to wrap an existing non-interactive element in your application’s UI.
+!!!
+
+#### Icon
+```handlebars
+  <Hds::TooltipButton @text="Hello!">
+    <FlightIcon @name="info" aria-label="More information" />
```

Comment:
```suggestion
    <FlightIcon @name="info" aria-label="More information" />
```

@KristinLBradley the aria-label should be on the button, not the FlightIcon.

Reply:
Updated, thanks!